tools/so-channeler/gha/README.md

```diff
@@ -0,0 +1,29 @@
+# StackOverflow to Slack GitHub Action
+
+This GitHub Action fetches new StackOverflow questions tagged with `open-telemetry` and posts them to a Slack channel. It runs on an hourly schedule.
+
+## Setup
+
+1. Clone this repository or create a new repository with the provided `github_action.py` script and `.github/workflows/main.yml` workflow file.
```

Comment:
Could we also just create the repo where this will be running in right away rather than having it in the community repo?
To separate the implementation from the state that's being kept it could just maintain this `state.txt` on a separate branch, similar to the gh-docs branches some repos use.